File: MyStagePass.Services/Database/Genre.cs
namespace MyStagePass.Services.Database
{
	public class Genre
	{
		public int GenreID {  get; set; }
		public string? Name { get; set; }
		public virtual ICollection<PerformerGenre> Performers { get; set; } = new List<PerformerGenre>();
	}
}

File: UI/mystagepass_admin/lib/models/Genre/genre.dart
import 'package:json_annotation/json_annotation.dart';

part 'genre.g.dart';

@JsonSerializable()
class Genre {
  @JsonKey(name: 'genreID')
  int? genreId;
  String? name;
  List<String>? performers;

  Genre({this.genreId, this.name, this.performers});

  factory Genre.fromJson(Map<String, dynamic> json) => _$GenreFromJson(json);
  Map<String, dynamic> toJson() => _$GenreToJson(this);
}
